src/core/gosling.schema.ts

```diff
@@ -49,38 +54,61 @@ export type Assembly = 'hg38' | 'hg19' | 'hg18' | 'hg17' | 'hg16' | 'mm10' | 'mm
 export type ZoomLimits = [number | null, number | null];
 
 export interface CommonViewDef {
+    /** specify the layout type of all tracks, either `"linear"` or `"circular"` */
     layout?: Layout;
+    /** either `"horizontal"` or `"vertical"` */
     orientation?: Orientation;
 
+    /**
+     * - if `{"layout": "linear"}`, specify the space between tracks in pixels;
+     *
+     * - if `{"layout": "circular"}`, specify the space between tracks in percentage ranging from 0 to 100.
+     */
     spacing?: number;
+    /** whether to disable [Zooming and Panning](http://gosling-lang.org/docs/user-interaction#zooming-and-panning),
+     * __Default:__ false.
```

Comment:
Do you think it is better to use code blocks like the below for all descriptions that are related to actual properties in the grammar (e.g., all default values)?

```suggestion
     * __Default:__ `false`.
```

Reply:
Agree. Will go through the code to check